presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -71,6 +71,7 @@
     public static final String ITERATIVE_OPTIMIZER_TIMEOUT = "iterative_optimizer_timeout";
```

Comment:
Hm, I was thinking about adding the round robin step. At least mentioning that there's a configuration you can set for that would be helpful for understanding what's up there.